Userland/Utilities/top.cpp

```diff
@@ -19,6 +20,23 @@
 #include <sys/ioctl.h>
 #include <unistd.h>
 
+struct TopOption {
+    enum class SortBy {
+        pid = 0,
+        tid,
+        priority,
+        username,
+        state,
+        virt,
+        phys,
+        cpu,
+        name
```

Comment:
Enum members should have PascalCase names.
and the initial value of `0` seems redundant.

Reply:
Better stated by https://github.com/SerenityOS/serenity/pull/8270#discussion_r659372796
Resolving this in favour of that.